connectivity/drivers/lora/COMPONENT_SX126X/mbed_lib.json

```diff
@@ -24,6 +28,17 @@
         "standby-mode": {
         	"help": "Default: STDBY_RC = 0, STDBY_XOSC = 1",
         	"value": 0
-        }
+        },
+        "spi-mosi":       { "value": "NC" },
```

Comment:
should this follow the style: value on separate line with NC ? same as the rest above to be consistent (I understand it's not much value but consistency is better.

Reply:
Done